### PR TITLE
Add OpenJDK 15

### PR DESCRIPTION
**Pull request by poad**

### CODE REVIEW

**Comment by Isabek** [comment]
Please, approve it. We need Java 15.

File: src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala [hunk 1]
```diff
@@ -160,4 +160,32 @@ class OpenJdkMigrations {
...
+            s"https://download.java.net/java/GA/jdk15/779bf45e88a44cbd9ea6621d33e33db1/36/GPL/$binary",
+            platform,
+            Some(OpenJDK)
+          )
+      }
+      .toList
+      .validate()
+      .insert()
+      .foreach { version =>
+        removeVersion("java", "14.0.2-open", version.platform)
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala:189]: Is this correct?

**Comment by marc0der** [comment]
@Isabek just waiting for one change then we can approve/merge.

File: src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala (hunk 1 above)
Comment (poad) [reply src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala:189]: I see!
I thought it was a replacement for the old GA, 14.0.2-open.
That's a mistake, and you are correct that it replaces 15.ea.36-open, which is Early Access for 15.
Correction.

**Comment by marc0der** [comment]
Thanks @poad!

**Comment by Isabek** [comment]
Thanks, @poad, and @marc0der.

**Pull request merged**